tutorial/src/App.css

```diff
@@ -27,18 +27,48 @@
 }
 
 .App-link {
-  color: #61dafb;
+  color: #e42023;
```

Comment:
We should probably define a color palette someplace.

Reply:
Good idea.